tests/Jmikola/Tests/WildcardEventDispatcher/WildcardEventDispatcherTest.php

```diff
@@ -191,4 +191,7 @@ public static function getSubscribedEvents()
             'core.multi' => [['onMulti1', 10], ['onMulti2', 20]],
         ];
     }
+
+    #[\ReturnTypeWillChange]
+    public function __call(string $name, array $arguments) {}
```

Comment:
Noted that this is required because EventDispatcherInterface specifies a `callable` hint its `$listener` arguments. That was introduced in 6.0 via https://github.com/symfony/event-dispatcher/commit/614aff36a26adc68fafdb048a46f42d6a05c9510 and later backported to 5.2.12+ in https://github.com/symfony/symfony/commit/6f6c1a1661a3861a4f0dbd07ac03f62dcd153fba.

Symfony's own test suite doesn't need `__call()` defined since it uses EventDispatcher directly; however, our tests uses a mock derived from EventDispatcherInterface so this is necessary.